LayoutTests/fast/forms/ios/initial-size-for-date-time-related-controls.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../../resources/js-test.js"></script>
+</head>
+<body onload="runTest()">
+<div id=parent>
+<input id='datetime-local' type='datetime-local'>
+<input id='date' type='date'>
+<input id='time' type='time'>
+<input id='month' type='month'>
+<input id='week' type='week'>
+</div>
+<pre id="description"></pre>
+<pre id="console"></pre>
+<script>
+function elementWidth(el) {
+    return el.getBoundingClientRect().right - el.getBoundingClientRect().x;
+}
+
+function elementHeight(el) {
+    return el.getBoundingClientRect().bottom - el.getBoundingClientRect().y;
+}
```

Comment:
Can we just use `clientHeight` and `clientWidth`?

https://developer.mozilla.org/en-US/docs/Web/API/Element/clientHeight